src/labels/CustomLabels.labels

```diff
@@ -2287,6 +2287,14 @@
         <shortDescription>Contains message indicating one installment has been selected to skip</shortDescription>
         <value>You&apos;ve selected 1 installment.</value>
     </labels>
+    <labels>
+        <fullName>RD2_PauseCannotExistOnScheduleChange</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Cannot change specific fields causing schedule change when a pause exists</shortDescription>
+        <value>You can&apos;&apos;t change the following fields when an active pause exists for a Recurring Donation: {0}.</value>
```

Comment:
@smarovich 
CC @force2b , @npsp-reedestockton 
Can you please review the custom label displayed when Recurring Donation cannot be updated due to an active pause schedule? The message is related to the AC6 of the [W-7807797](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008Sr8EIAS/view).

Only specific fields are considered: "Day of Month", 'Installment Period" and "Installment Frequency". One or all of them can change. I think it is valuable to display to the user which fields cannot be changed. 

If the user would need to update the RD, they need to deactivate the active pause, update the RD and then re-create the pause. Was not sure how to word the message with instructions.

Here are screenshots:
<img width="924" alt="Screen Shot 2020-08-10 at 8 34 39 PM" src="https://user-images.githubusercontent.com/28743286/89861836-89492300-db5b-11ea-886d-49f0c5f2749e.png">

<img width="917" alt="Screen Shot 2020-08-10 at 8 34 16 PM" src="https://user-images.githubusercontent.com/28743286/89861844-8e0dd700-db5b-11ea-99af-5911a904c028.png">

<img width="925" alt="Screen Shot 2020-08-10 at 8 33 56 PM" src="https://user-images.githubusercontent.com/28743286/89861854-936b2180-db5b-11ea-9c11-d41dbf385686.png">

<img width="926" alt="Screen Shot 2020-08-10 at 8 33 31 PM" src="https://user-images.githubusercontent.com/28743286/89861866-96661200-db5b-11ea-83b8-084e12d05799.png">

Reply:
I think it might be better for the message to refer to the pause range somehow.  Maybe something like:

"You can't change the following fields when the new schedule includes paused installments".  @smarovich can do much better than that, but you get the idea I hope.

Reply:
@RadSFDeveloper What exactly is paused? The Recurring Donation? The installments? The schedule?
Here are two options. Happy to revise as needed!

You can't update the following field(s) because this Recurring Donation is paused: <field>, <field>. If you'd like to update the field(s), you must deactivate the pause. 
OR 
You can't update the following field(s) when a Recurring Donation schedule includes paused installments: <field>, <field>. If you'd like to update the field(s), you must deactivate the pause.

Reply:
Thank you @smarovich ! I think the first option works great.

Recurring Donation can be paused when at least one installment is skipped. At most 12 installments can be skipped. If the skipped installments are in past, then Recurring Donation does not have an active pause and certain fields can be edited. If the Recurring Donation has current or future installments that are going to be skipped, then the pause is active and certain fields cannot be changed unless the pause is deactivated (removed).

The error on the Recurring Donation edit form:
<img width="917" alt="Screen Shot 2020-08-11 at 11 18 53 AM" src="https://user-images.githubusercontent.com/28743286/89933858-8b939780-dbc4-11ea-9cc3-2b6edd8ed13e.png">

Example of the Pause Recurring Donation modal:
<img width="1750" alt="Screen Shot 2020-08-11 at 11 22 44 AM" src="https://user-images.githubusercontent.com/28743286/89934149-0066d180-dbc5-11ea-9fee-7cb98f2eb88b.png">

Reply:
Looks good!